src/cinder/app/cocoa/AppImplMac.mm

```diff
@@ -628,9 +628,21 @@ - (void)resize
 	NSSize nsSize = [mCinderView frame].size;
 	mSize = cinder::ivec2( nsSize.width, nsSize.height );
 
+	NSRect frame = [mWin frame];
+	NSRect content = [mWin contentRectForFrameRect:frame];
+	
+	ivec2 prevPos = mPos;	
+	mPos = ivec2( content.origin.x, [[[NSScreen screens] objectAtIndex:0] frame].size.height - frame.origin.y - content.size.height );
```

Comment:
Hmm, doesn't this assume that the window is on the first screen? It might be a better idea to get the screen from the window's `screen` property and the `NSGeometry.h` functions might be helpful too:
```Objective-C
mPos = ivec2( NSMinX( content ), NSHeight( mWin.screen.frame ) - NSMinY( frame ) - NSHeight( content ) );
```